vlm/src/main/scala/geotrellis/contrib/vlm/gdal/GDALReprojectRasterSource.scala

```diff
@@ -67,8 +68,8 @@ case class GDALReprojectRasterSource(
   }
 
   override def reproject(targetCRS: CRS, options: Reproject.Options): RasterSource =
-    GDALReprojectRasterSource(uri, targetCRS, options, warpList, alignTargetPixels)
+    GDALReprojectRasterSource(uri, targetCRS, options, alignTargetPixels, warpList)
```

Comment:
Missed this in the last PR but shouldn't `alignTargetPixels` be part of the `options` ?
Does it make things easier to break it out ?

Reply:
It can, but there appears a question: does it make any sense? in some operations (resample, reprojection) we calculate target cellSize basing on the the raster params / smth else, and these options should override / partially override these calculated values?